gulpfile.js

```diff
@@ -0,0 +1,22 @@
+var gulp = require('gulp');
+var jshint = require('gulp-jshint');
+require('jshint-stylish');
+var mocha = require('gulp-mocha');
+
+gulp.task('run-tests', ['lint'], function() {
+  gulp.src(['test/*.js'], { read: false })
+    .pipe(mocha());
+});
+
+gulp.task('watch-files', function() {
+  gulp.watch(['./*.js', 'test/**/*.js', 'models/*.js', 'public/**/*.js', 'routes/*.js'], ['run-tests']);
+});
+
+gulp.task('lint', function() {
+  gulp.src(['./*.js', 'test/**/*.js', 'routes/*.js', 'models/*.js'])
+    .pipe(jshint('.jshintrc'))
```

Comment:
The `.jshintrc` file is not in the project. Had to comment this line out to get tests to run.